src/script/api/script_event.hpp

```diff
@@ -32,6 +32,7 @@ class ScriptEvent : public ScriptObject {
 		ET_SUBSIDY_EXPIRED,
 		ET_ENGINE_PREVIEW,
 		ET_COMPANY_NEW,
+		ET_COMPANY_RENAMED,
```

Comment:
Inserting it here should not cause issues, but it might be better to put it at the end of the enum.